CMakeLists.txt

```diff
@@ -140,7 +152,8 @@ configure_file(
 
 ################################################################
 
-if (NOT APPLE)
+if (NOT APPLE AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS "9.0")
```

Comment:
You also need to check CMAKE_CXX_COMPILER_ID STREQUAL "GNU"

Reply:
Actually it should be 9.1 per the notes in https://en.cppreference.com/w/cpp/filesystem

Reply:
ok, I made the requested changes. Currently running on Jenkins. Should be good to go.
Thx for your replies